web/src/compose.js

```diff
@@ -186,7 +186,7 @@ export function clear_compose_box() {
     $("#compose-textarea").removeData("draft-id");
     compose_ui.autosize_textarea($("#compose-textarea"));
     compose_banner.clear_errors();
-    compose_banner.clear_warnings();
+    compose_banner.clear_all();
```

Comment:
if you do come up with a generic name for a function to clear the right set of errors, make sure to remove `clear_errors` here :)